pkg/test/step_integration_test.go

```diff
@@ -47,7 +51,7 @@ func TestCheckResourceIntegration(t *testing.T) {
 		{
 			testName: "match object by labels, first in list matches",
 			actual: []runtime.Object{
-				testutils.WithSpec(testutils.WithLabels(testutils.NewPod("aa", ""), map[string]string{
+				testutils.WithSpec(testutils.WithLabels(testutils.NewPod("labels-match-pod", ""), map[string]string{
```

Comment:
Lol, where have I seen these "speaking pod names"? 🤔

Reply:
:D 🤔 anyway the error is that pod 'aa' already exists :D 🤷‍♀

Reply:
We should make all pod names properly unique. Something along the lines of:
```go
func PodName(suiteName, testName, podName string) {
  reg, err := regexp.Compile("[^a-zA-Z0-9]+")
  if err != nil {
      log.Fatal(err)
  }
  return strings.ToLower(reg.ReplaceAllString(fmt.Sprintf("%s-%s-%s", suiteName, testName, podName), ""))
}
```
and call it:
```go
testutils.NewPod(PodName(t.Name, testName, "labels-match-pod"))
```

Reply:
@zen-dog I kind of like them hand-picked based on the name of the test... it makes it easier for debugging as long as you're not generating 50 pods or so per test...